tests/integration/test_basic.py

```diff
@@ -63,6 +64,7 @@ def foo(x: bool) -> bool:
     validate(foo)
 
 
+@pytest.mark.xfail(reason="hugr-includes-whole-stdlib")
```

Comment:
I did try fixing this one, see commit b663e8c5144a21c6e09aea2a7b1c144af7e3c4ab. But gave up when I realized how many other problems there were, so reverted and xfail this too to be consistent...

Reply:
Yeah, xfail is fine for this PR. Can you add an issue so we don't forget about it?

> The "right" solution - while we wait for proper Hugr linking - might well be to add metadata to every Hugr FuncDefn saying which source file it's from so we can filter better....but that's another PR? Or is there something like this already?

This doesn't exist yet, but sounds like good idea 👍